### PR TITLE
Add support for deploying kubernetes ui after installing kubernetes m…

### PR DESCRIPTION
**Pull request by mabuaisha**
…anager

### CODE REVIEW

File: examples/kubernetes-dashboard-blueprint.yaml [hunk 1]
```diff
@@ -0,0 +1,342 @@
...
+
+outputs:
+
+  bearer_token:
+    description: Kubernetes Token
+    value: { get_attribute: [dashboard_token_and_port, bearer_token] }
+
+  dashboard_port:
+    description: Kubernetes Access Port
+    value: { get_attribute: [dashboard_token_and_port, dashboard_port] }
```
Comment (EarthmanT) [inline examples/kubernetes-dashboard-blueprint.yaml]: Let's change this to:
```yaml
    value: { concat: [ "https://", { get_input: public_dashboard_ip }, ":", { get_attribute: [dashboard_token_and_port, dashboard_port] } ] }
```

File: examples/kubernetes-dashboard-blueprint.yaml [hunk 2]
```diff
@@ -0,0 +1,342 @@
...
+          client-key-data:  { get_input: kubernetes-admin_client_key_data }
+
+  kubernetes_configuration_file_content:
+    description: >
+      File content of kubernetes master YAML configuration
+    default: { get_input: kubernetes_master_configuration }
+
+  public_dashboard_ip:
+    description: >
+      The public ip of the instance where dashboard is running
```
Comment (EarthmanT) [inline examples/kubernetes-dashboard-blueprint.yaml]: "of the Kubernetes Master"

File: README.md [hunk 3]
```diff
@@ -176,3 +176,17 @@ As *authentication* property of Master node you can specify dictionary with key
...
+
+
+### Upload Kubernetes Dashboard UI Blueprint To Manager
+```shell
+
+1. Update dashboard input file ***dashboard_input.yaml***
+2. Install the dashboard: cfy install -b kubernetes-dashboard -n kubernetes-dashboard examples/dashboard.yaml -i examples/inputs/dashboard-inputs.yaml
+3. Run the following command: cfy deployment outputs kubernetes-dashboard
+4. The output of deployment command should generate ***dashboard_port*** &&  ***bearer_token***
+5. Access the Dashboard using ***https://MASTER_IP:{{dashboard_port}}***
```
Comment (EarthmanT) [inline README.md]: Add here a step to open up the Port on your security group if it is not already open.

File: examples/kubernetes-dashboard-blueprint.yaml (hunk 1 above)
Comment (mabuaisha) [reply examples/kubernetes-dashboard-blueprint.yaml]: @EarthmanT done

File: examples/kubernetes-dashboard-blueprint.yaml (hunk 2 above)
Comment (mabuaisha) [reply examples/kubernetes-dashboard-blueprint.yaml]: @EarthmanT done

File: README.md (hunk 3 above)
Comment (mabuaisha) [reply README.md]: @EarthmanT done

**Pull request merged**